packages/data/src/redux-store/index.js

```diff
@@ -700,22 +700,15 @@ function mapSelectorWithResolver(
 	boundMetadataSelectors
 ) {
 	function fulfillSelector( args ) {
-		const state = store.getState();
-
 		if (
 			resolversCache.isRunning( selectorName, args ) ||
-			( typeof resolver.isFulfilled === 'function' &&
-				resolver.isFulfilled( state, ...args ) )
-		) {
-			return;
-		}
-
-		if (
 			boundMetadataSelectors.hasStartedResolution( selectorName, args )
 		) {
 			return;
 		}
 
+		const state = store.getState();
```

Comment:
This could be inlined into the `isFulfilled` call, we don't use the `state` anywhere else do we?
```js
resolver.isFulfilled( store.getState(), ...args )
```